query/src/main/java/io/keen/client/java/Query.java

```diff
@@ -36,8 +42,8 @@
      *
      * @return The JSON object map.
      */
-    public Map<String, Object> constructQueryArgs() {
-
+    @Override
+    Map<String, Object> constructRequestArgs() {
```

Comment:
Ditto. Similar to responses elsewhere, the fact that one can't imagine client code calling it implies it shouldn't really have been callable from another package to start with. So the question is this: how strictly we should adhere to semver here and in similar probably-shouldn't-have-been-that-way situations?